app/src/main/java/org/mozilla/tv/firefox/settings/SettingsFragment.kt

```diff
@@ -34,11 +34,15 @@ class SettingsFragment : Fragment() {
 
         ic_lock.setImageResource(R.drawable.mozac_ic_lock)
         telemetryButton.isChecked = DataUploadPreference.isEnabled(context)
-        telemetryView.setOnClickListener { _ ->
+        val dataPreferenceClickListener = { _: View ->
             val newTelemetryState = !DataUploadPreference.isEnabled(context)
             DataUploadPreference.setIsEnabled(context, newTelemetryState)
             telemetryButton.isChecked = newTelemetryState
         }
+        // Due to accessibility hack for #293, either of these views could be unfocusable, so we
+        // need to set the click listener on both.
+        telemetryView.setOnClickListener(dataPreferenceClickListener)
+        telemetryButton.setOnClickListener(dataPreferenceClickListener)
```

Comment:
Nit: so, one of these views is displayed when Talk Back is enabled and the other when it's not?  If that understanding is correct, could we rename them to better describe that?

Reply:
Good point that this is confusing - both are always visible, but `telemetryView` is actually the container. It's not that they're not "displayed", it's that one is focusable and the other is not, so that VoiceView can select the correct one to read aloud. They'll both be visible, all the time.

I'll update the naming to make this more clear, and the comment too.